packages/flutter_localizations/lib/src/l10n/date_localizations.dart

```diff
@@ -6779,10 +6779,10 @@ const Map<String, dynamic> dateSymbols = <String, dynamic>{
       r'''S'''
     ],
     'SHORTQUARTERS': <dynamic>[
-      r'''1. cet.''',
-      r'''2. cet.''',
-      r'''3. cet.''',
-      r'''4. cet.'''
+      r'''1. cet.''',
```

Comment:
What's this change here?

Reply:
I'm not sure. The old and new versions of these four lines are identical.